custom_fortunes/computer-programmer_humor

```diff
@@ -0,0 +1,7 @@
+"There's no place like ~"
+    - Dorothy Gale
+%
+In order to understand recursion you must first understand recursion.
+%
+42
+
```

Comment:
I'm guessing `42` is meant to be another quote in here? Looks like we need to add the `%` sign on line 7.  While we're at it, could we attribute it to an author or the work that it comes from?

Reply:
Works either way I guess. The original FORTUNE has loads of unquoted jokes / sayings, and it's not like a single number is exactly a quote.
Maybe we could quote Deep Thought ;) that'd be funnier.

Reply:
Hmm ... sorry, I don't quite understand
(I will add a Deep Thought quote anyway)

Reply:
I see you merged this PR. Should I make a new one for the `42` stuff or is it not a big deal?

Reply:
I think it's fine right now. But it's your programmer_humour @DarthFloopy , so of course make changes or additions as you see fit in a new PR

Reply:
@heptocat Right, but most of those have authors attributed to them when needed. In this case, 42 is a pretty well-known reference to Hitchhiker's Guide. I can see quality of these going really down if we don't at least loosely try to ensure people add sources to quotes when needed, especially when they are short.

Reply:
I had actually marked this as needing a revision for that reason.